### PR TITLE
Updating workflows, add codeql, add questions

### PR DESCRIPTION
**Pull request by zemberdotnet**
# Description

This PR updates workflows in `graph-jamf`, adds the `codeql-analysis` workflow, and `questions` workflow. This is a test for automatic syncing between the `integration-template` and `graph-*` projects.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -71,4 +65,29 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > .npmrc
           yarn
-          npm publish
+          yarn build
+          npm publish ./dist
+
```
Comment (zemberdotnet) [inline .github/workflows/build.yml:70]: I'm concerned about this new publish.

Comment (zemberdotnet) [reply .github/workflows/build.yml:70]: I think the `package.json` `build` command or `files` property will have to be updated if this change was made.

**Review by mdaum — APPROVED** [review]
Approved. Let's verify that the managed integration is working via manual dev deploy before using this new version. Thanks!

**Review by adam-in-ict — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: yarn
+
```
Comment (VDubber) [inline .github/workflows/codeql-analysis.yml:34]: Can we get 
```
- name: Test
   run: yarn test:ci
```
included as well?

Comment (VDubber) [reply .github/workflows/codeql-analysis.yml:34]: I guess this would be included in the build.yml file.

**Review by VDubber — APPROVED** [review]

**Pull request merged**